RecoLocalMuon/CSCValidation/src/CSCValidation.cc

```diff
@@ -859,18 +860,10 @@ void CSCValidation::doPedestalNoise(edm::Handle<CSCStripDigiCollection> strips)
       int myStrip = pedIt->getStrip();
       std::vector<int> myADCVals = pedIt->getADCCounts();
       float TotalADC = getSignal(*strips, id, myStrip);
-      bool thisStripFired = false;
       float thisPedestal = 0.5 * (float)(myADCVals[0] + myADCVals[1]);
       float thisSignal =
           (1. / 6) * (myADCVals[2] + myADCVals[3] + myADCVals[4] + myADCVals[5] + myADCVals[6] + myADCVals[7]);
-      float threshold = 13.3;
-      if (id.station() == 1 && id.ring() == 4) {
```

Comment:
@ptcox I'm puzzled about this dead reassignment of `myStrip`: was it intended to be somewhere else (e.g. above)?